table/table_test.cc

```diff
@@ -1189,7 +1189,7 @@ TEST_F(TablePropertyTest, PrefixScanTest) {
                                 {"num.555.3", "3"}, };
 
   // prefixes that exist
-  for (const std::string& prefix : {"num.111", "num.333", "num.555"}) {
+  for (const std::string prefix : {"num.111", "num.333", "num.555"}) {
```

Comment:
revert these changes.

Reply:
It will fail caused by `-Wrange-loop-analysis`, so I changed it.